src/lib/database.js

```diff
@@ -24,6 +24,12 @@ const RANGE_TRANSLATION = {
   '>': 'gt',
   '>=': 'gte'
 }
+const UNPREFIXED_FIELDS = [
+  'id',
+  'collection',
+  'geometry',
+  'bbox'
```

Comment:
I would remove bbox for now, as this is an array, and there's no support yet for the Array Function conformance class -- so there's no way to construct a clause that uses it.

Reply:
Removed. Is `geometry` also be a problem? When present in an incoming query it will be parsed as an object (so not one of cql2's scalar data types). Even if we converted an incoming query geometry field to a string,  geometries stored in OpenSearch are of the geoshape type, which I don't think you can search using OpenSearch DSL expressions like `term` or `range`.

Reply:
That's a good point. I reverse what I said -- I think we should include all four as non-prefixed, in anticipation of future support for bbox (via Array Functions) and geometry (via Basic Spatial Operations).  We should document (I think this is already in the README?) about them not being prefixed, and then note explicitly that `bbox` and `geometry` aren't prefixed, but also can't actually be searched over right now until those other conformance classes are implemented.

Reply:
also, you're right that you can't use term or range with geoshape, you have to use geoshape query: https://opensearch.org/docs/latest/query-dsl/geo-and-xy/geoshape/

Reply:
Added back `bbox`, added note about unprefixed top-level Item fields, and added note about `bbox` and `geometry` search not supported until additional conformance classes are implemented.